### ISSUE TITLE
Custom dataset training

### ISSUE THREAD

**Issue by willieChen58** [zju3dv/clean-pvnet#291]
Hello, I noticed that when training my custom dataset the training set and the test/validation set are the same, so I wanted to ask in which python file can I change the code to make the training set and the test/validation set different?

**Comment by nn18339193656** [comment]
你好 ，我也想解决这个问题，请问你解决了吗

**Comment by willieChen58** [comment]
> 你好 ，我也想解决这个问题，请问你解决了吗

我把handle_custom_dataset.py这个文件重新改写了一下问题才解决的，这几天我有空发一个博客吧

**Comment by nn18339193656** [comment]
好的好的 谢谢大佬

**Comment by willieChen58** [comment]
> 好的好的 谢谢大佬

方便的话加一下我的qq吧，2392277518。

**Comment by nn18339193656** [comment]
好的 已加



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年01月12日 16:53 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [zju3dv/clean-pvnet] Custom dataset training (Issue #291) |

好的好的 谢谢大佬

方便的话加一下我的qq吧，2392277518。
